community/feature-status.md

```diff
@@ -0,0 +1,156 @@
+# Feature Status
+
+This document describes the different statuses a given feature can
+have in Argo projects.
+
+## Overview
+
+Argo umbrella projects are maintained by the open-source community.
+This means that features are provided by different organizations with
+different use-cases. While certain features are battle tested by
+corporations with large infrastructure, this is not always the case.
+Feature status is a classification that can be used by Argo projects
+to clarify and set expectations about a given functionality. However,
+not every feature will have to be associated with a given status.
+Maintainers will use this classification as a framework and gradually
+associate features with the proper current status. When available, the
+current status will be informed in the documentation of that specific
+feature.
+
+## Definition
+
+Argo projects adopt the standard software life cycle terminology to
+define the different feature statuses:
+
+- _Alpha_
+- _Beta_
+- _Stable_
+
+### Alpha
+
+_Alpha_ means that a feature is in its early stage and is considered
+as experimental which means that it can be removed from future
+releases. This is an useful status to use when there are uncertainties
+about a new feature and maintainers want to get feedback from the
+community. _Alpha_ features are usually behind feature flags disabled
+by default. Users are expected to understand the risks of enabling
+_Alpha_ features.
+
+### Beta
+
+_Beta_ features are usually enabled by default and are not considered
+as experimental. _Beta_ features are considered in its early maturity
+stage. This means that common use cases are functional but there are
+known uncovered edge cases. Edge cases will be tracked as issues and
+are expected to be voted on. Votes take place by adding a thumbs up
+emoji reaction in the main description of an issue. In most cases,
+issues with more votes will take higher priority. _Beta_ features may
+not be backwards compatible with previous releases while in this
+status.
```

Comment:
To make clear that beta features aren't allowed to break backwards-compatibilty _in general_, just within the feature itself.

```suggestion
issues with more votes will take higher priority. _Beta_ features may
not be backwards compatible with previous releases of the feature
while in this status.
```

Reply:
changed. Tks!